src/app/component/Router/Router.container.js

```diff
@@ -0,0 +1,179 @@
+        );
+        ConfigDispatcher.then(
+            /** @namespace Component/Router/Container/then */
+            ({ default: dispatcher }) => dispatcher.handleData(dispatch)
+        );
+        HeaderAndFooterDispatcher.then(
+            /** @namespace Component/Router/Container/then */
+            ({ default: dispatcher }) => dispatcher.handleData(dispatch, options)
+        );
+    }
+});
+
+/** @namespace Component/Router/Container */
+export class RouterContainer extends PureComponent {
+    static propTypes = {
+        init: PropTypes.func.isRequired,
+        updateMeta: PropTypes.func.isRequired,
+        default_description: PropTypes.string,
+        default_keywords: PropTypes.string,
+        default_title: PropTypes.string,
+        title_prefix: PropTypes.string,
+        title_suffix: PropTypes.string,
+        isLoading: PropTypes.bool,
+        isBigOffline: PropTypes.bool
+    };
+
+    static defaultProps = {
+        default_description: '',
+        default_keywords: '',
+        default_title: '',
+        title_prefix: '',
+        title_suffix: '',
+        isLoading: true,
+        isBigOffline: false
+    };
+
+    __construct(props) {
+        super.__construct(props);
+
+        this.initializeApplication();
+    }
+
+    componentDidUpdate(prevProps) {
+        const { isLoading, updateMeta } = this.props;
+        const { isLoading: prevIsLoading } = prevProps;
+
+        if (!isLoading && isLoading !== prevIsLoading) {
+            const {
+                default_description,
+                default_keywords,
+                default_title,
+                title_prefix,
+                title_suffix
+            } = this.props;
+
+            updateMeta({
+                default_title,
+                title: default_title,
+                default_description,
+                description: default_description,
+                default_keywords,
+                keywords: default_keywords,
+                title_prefix,
+                title_suffix
+            });
+        }
+    }
+
+    containerProps = () => {
+        const { isBigOffline } = this.props;
+
+        return { isBigOffline };
+    };
+
+    initializeApplication() {
+        const { init } = this.props;
+        init(this.getHeaderAndFooterOptions());
+    }
+
+    getCmsBlocksToRequest() {
+        // TODO: remove!
+
+        const blocks = Object.values(window.contentConfiguration).reduce(
+            (acc, config) => [
+                ...acc,
+                ...Object.entries(config).reduce(
+                    (acc, [key, identifier]) => ((key.indexOf('cms') === -1)
+                        ? acc
+                        : [...acc, identifier]
+                    ),
+                    []
+                )
+            ],
+            []
+        ).filter((value, index, self) => value && self.indexOf(value) === index);
+
+        return blocks.length ? blocks : ['social-links'];
+    }
```

Comment:
Check the current index.js this logic is not present there

Reply:
Removed this